portal-frontend/src/app/features/notice-of-intents/edit-submission/proposal/pofo/pofo-proposal.component.ts

```diff
@@ -98,8 +96,7 @@ export class PofoProposalComponent extends FilesStepComponent implements OnInit,
           followUpIds: noiSubmission.soilFollowUpIDs,
           purpose: noiSubmission.purpose,
           soilFillTypeToPlace: noiSubmission.soilFillTypeToPlace,
-          projectDurationAmount: noiSubmission.soilProjectDurationAmount?.toString() ?? null,
-          projectDurationUnit: noiSubmission.soilProjectDurationUnit,
+          projectDuration: noiSubmission.soilProjectDuration?.toString() ?? null,
```

Comment:
This doesn't need `?.toString() ?? null` because it is already a string input

Reply:
changed